docs/fundamentals/runtime-libraries/snippets/System/Boolean/Overview/csharp/binary1.cs

```diff
@@ -10,12 +10,11 @@ public static void Main()
         {
             // Get binary representation of flag.
             Byte value = BitConverter.GetBytes(flag)[0];
-            Console.WriteLine("Original value: {0}", flag);
-            Console.WriteLine("Binary value:   {0} ({1})", value,
-                              GetBinaryString(value));
+            Console.WriteLine($"Original value: {flag}");
+            Console.WriteLine($"Binary value:   {value} ({GetBinaryString(value)})");
             // Restore the flag from its binary representation.
             bool newFlag = BitConverter.ToBoolean(new Byte[] { value }, 0);
-            Console.WriteLine("Restored value: {0}\n", flag);
+            Console.WriteLine($"Restored value: {flag}\n");
```

Comment:
nit: to be truly cross-plat, consider `Environment.NewLine` instead of `\n`:

```suggestion
            Console.WriteLine($"Restored value: {flag}{Environmemt.NewLine}");
```

Reply:
```suggestion
            Console.WriteLine($"Restored value: {flag}{Environment.NewLine}");
```